runtime/onert/core/include/ir/operation/RoPE.h

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __ONERT_IR_OPERATION_ROPE_H__
+#define __ONERT_IR_OPERATION_ROPE_H__
+
+#include "ir/Operation.h"
+#include "ir/InternalType.h"
```

Comment:
May I ask why this header is included?

Reply:
i added it wrong by mistake
i will delete it.
thanks